app/src/main/java/org/jellyfin/mobile/player/ExoPlayerFormats.kt

```diff
@@ -271,6 +271,23 @@ object ExoPlayerFormats {
             else -> null
         }
     }
+
+    fun getAudioProfile(codec: String, profile: Int): String? =  when(codec) {
+        "aac" -> getAACProfile(profile)
+        else -> null
+    }
+
+    private fun getAACProfile(profile: Int): String? = when(profile) {
+        MediaCodecInfo.CodecProfileLevel.AACObjectELD -> "ELD"
```

Comment:
Maybe add an import for `MediaCodecInfo.CodecProfileLevel.*`

Reply:
@nielsvanvelzen I'm not sure if that's such a good idea. I think an import for `MediaCodecInfo.CodecProfileLevel` would be enough, requiring to write e.g. `CodecProfileLevel.AACObjectELD` afterwards.